mpas_analysis/sea_ice/time_series.py

```diff
@@ -260,25 +260,6 @@ def run_task(self):  # {{{
         timeEnd = date_to_days(year=yearEnd, month=12, day=31,
                                calendar=calendar)
 
-        if preprocessedReferenceRunName != 'None':
-            inFilesPreprocessed = '{}/icevol.{}.year*.nc'.format(
-                preprocessedReferenceDirectory, preprocessedReferenceRunName)
-            dsPreprocessed = open_multifile_dataset(
-                fileNames=inFilesPreprocessed,
-                calendar=calendar,
-                config=config,
-                timeVariableName='xtime')
-            preprocessedYearEnd = days_to_datetime(dsPreprocessed.Time.max(),
-                                                   calendar=calendar).year
-            if yearStart <= preprocessedYearEnd:
-                dsPreprocessedTimeSlice = \
-                    dsPreprocessed.sel(Time=slice(timeStart, timeEnd))
-            else:
-                self.logger.warning('Preprocessed time series ends before the '
-                                    'timeSeries startYear and will not be '
-                                    'plotted.')
-                preprocessedReferenceRunName = 'None'
-
```

Comment:
I am looking at the changes more closely, to try to understand what is going on in my weird ice timeseries error.
So, I noticed this piece above: is it being replaced by something else?

Reply:
I think it's totally redundant to the code below.  Maybe I'm missing something.

Reply:
I see.  Maybe what this *does* accomplish is not plotting the reference data if it's out of range.  And now it tries to plot it even if it's out of range?  Maybe your case tests that (because your years are way later than mine)?

Reply:
I guess I am looking at the piece starting with `if  yearStart`. I remember we put that in there for when the preprocessed timeseries ended before the requested endYear.

Reply:
Let me add this piece back in (with the appropriate changes) and we'll see if it fixes your problem.

Reply:
yes, exactly, I am running with years=495, I think. way past when the preprocessed data are available. Good catch (both of us :)